src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -236,25 +243,24 @@ static void uct_rdmacm_cm_handle_event_connect_response(struct rdma_cm_event *ev
                   ucs_sockaddr_str(remote_addr, ip_port_str,
                                    UCS_SOCKADDR_STRING_LEN));
         uct_rdmacm_cm_ep_error_cb(cep, &remote_data, status);
-        return;
+        goto out;
```

Comment:
why invoke rdma_establish() in case of an error?

Reply:
should we call `rdma_disconnect` (or `rdamcm_reject` ??) from client side to notify server about error?